openslides/mediafiles/projector.py

```diff
@@ -16,11 +16,9 @@ class MediafileSlide(ProjectorElement):
     def get_context(self):
         pk = self.config_entry.get('id')
         try:
-            mediafile = Mediafile.objects.get(pk=pk)
+            Mediafile.objects.get(pk=pk)
         except Mediafile.DoesNotExist:
             raise ProjectorException(_('File does not exist.'))
-        if not (mediafile.is_presentable and mediafile.filetype == 'application/pdf'):
```

Comment:
I think we agreed that the client handles this. The server now only saves the files.